packages/core/src/extensions/Blocks/api/block.ts

```diff
@@ -151,8 +151,20 @@ export function createBlockSpec<
       return ({ HTMLAttributes, getPos }) => {
         // Create blockContent element
         const blockContent = document.createElement("div");
-        // Sets blockContent class
-        blockContent.className = styles.blockContent;
+        // Add custom HTML attributes
+        const blockContentDOMAttributes =
```

Comment:
why not blockContentDOMAttributes.blockContent.class?

Reply:
For each of the DOM element types, you can add custom attributes, not just a custom class

Reply:
Ok, you could do this:

constr { class, ...otherAttributes} = this.options.domAttributes?.blockContent || {};

Reply:
Actually this also doesn't really work since `class` is a reserved word for class definitions

Reply:
Check. We could decide to rename `class` to `className`, that's also inline with how React does this

Reply:
Not sure if it's a good idea tbh, since we're not only applying the class to React components. Basic nodes like `blockContainer`, `blockGroup`, etc, are defined outside of React, so I think it's better to keep HTML naming conventions for this rather than React ones